packages/adblocker/src/data-view.ts

```diff
@@ -20,6 +20,9 @@ export const EMPTY_UINT32_ARRAY = new Uint32Array(0);
 // Check if current architecture is little endian
 const LITTLE_ENDIAN: boolean = new Int8Array(new Int16Array([1]).buffer)[0] === 1;
 
+// TextEncoder doesn't need to be recreated every time unlike TextDecoder
+const TEXT_ENCODER = new TextEncoder();
```

Comment:
alternatively you could do:
```
const encoder = new TextEncoder();
const encode = encoder.encode.bind(encoder);
```

Reply:
`encode` is already exported by `punycode`. Would you like to have another variable name?

Reply:
We use this way in smaz. Closing.